source/postprocess/visualization.cc

```diff
@@ -446,30 +446,30 @@ namespace aspect
       if (output_format=="hdf5")
         {
           XDMFEntry new_xdmf_entry;
-          std::string     h5_solution_file_name = this->get_output_directory() + "solution/" + solution_file_prefix + ".h5";
-          std::string     xdmf_filename = this->get_output_directory() + "solution.xdmf";
+          const std::string h5_solution_file_name = "solution/" + solution_file_prefix + ".h5";
```

Comment:
I am unable to see from this patch, what actually changed, is it that ``h5_solution_file_name`` doesn't have the output directory in there?